GameData/ROCapsules/PartConfigs/Apollo/ApolloSM.cfg

```diff
@@ -7,9 +7,9 @@ PART
 	RSSROConfig = true
 	RP0conf = true
```

Comment:
Do not set RP0 configuration state.
As best I can tell, there are 243 lines of whitespace changes, and no content change.